specs/eip4844/polynomial-commitments.md

```diff
@@ -165,6 +168,20 @@ def bytes_to_bls_field(b: Bytes32) -> BLSFieldElement:
     return BLSFieldElement(field_element)
 ```
 
+
+#### `validate_kzg_g1`
+
+```python
+def validate_kzg_g1(b: Bytes48) -> None:
+    """
+    Convert 48-byte value to a G1 point that can be used to hold a KZGCommitment or KZGProof.
+    This performs the validation specified for the types above.
+    """
```

Comment:
Not sure if it's just GitHub or what, but why is "above" wrapped here?

Reply:
Doesn't look wrapped to me.

Reply:
Yeah nevermind. This is fine. Just a bug on GitHub or something.